src/core/lib/promise/party.cc

```diff
@@ -288,9 +286,11 @@ bool Party::RunOneParticipant(int i) {
             std::string(name).c_str(), i);
   }
   // Poll the participant.
+  gpr_log(GPR_INFO, "Begin PollParticipantPromise %p:%d", &sync_, i);
```

Comment:
Do we want these unconditional log statements?

Reply:
yeah no (fixed)